### ISSUE TITLE
Provide builds for windows 7

### ISSUE THREAD

**Issue by libreom** [Alex313031/IceCat#1]
Thanks

**Comment by Alex313031** [comment]
@libreom Yeah, since IceCat is based on ESR, it already supports win7 out of the box. Unfortunately, part of GNUs thing is using their own toolchain, instead of "bootstrapping" from Mozilla. What this means is that it can't be cross compiled for Windows on Linux, so Ill have to boot into my Win10 drive to build it. Probably will do that when I make the next mercury releases, and just build all the firefox stuff in one full sweep.
